openweathermap-ios/openweathermap-ios/BookmarkLocation/Model/Bookmark.swift

```diff
@@ -0,0 +1,5 @@
+import Foundation
+
+enum Bookmark: String {
```

Comment:
It's waste the usage of the word **Bookmark** just to be your archive key. Consider declaring it with the manager and let it be private.

Reply:
Fixed in 212f90e